<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>enterPoReturn</class>
 <widget class="QWidget" name="enterPoReturn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Purchase Order Returns</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="_2">
         <property name="leftMargin">
          <number>14</number>
         </property>
         <item>
          <widget class="OrderCluster" name="_po">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QCheckBox" name="_dropShip">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Drop Ship</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="XCheckBox" name="_printReport">
         <property name="text">
          <string>Print Return Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_returnAddrLit">
         <property name="text">
          <string>Return Address:</string>
         </property>
         <property name="buddy">
          <cstring>_returnAddr</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AddressCluster" name="_returnAddr">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="_close">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_post">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Post</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="_poitemLit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Purchase Order Items:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="XTreeWidget" name="_poitem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_enter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Enter Return</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>AddressCluster</class>
   <extends>QWidget</extends>
   <header>addresscluster.h</header>
  </customwidget>
  <customwidget>
   <class>OrderCluster</class>
   <extends>QWidget</extends>
   <header>ordercluster.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_printReport</tabstop>
  <tabstop>_returnAddr</tabstop>
  <tabstop>_poitem</tabstop>
  <tabstop>_enter</tabstop>
  <tabstop>_post</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>enterPoReturn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_poitem</sender>
   <signal>itemSelected(int)</signal>
   <receiver>_enter</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_poitem</sender>
   <signal>valid(bool)</signal>
   <receiver>_enter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_printReport</sender>
   <signal>toggled(bool)</signal>
   <receiver>_returnAddr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_po</sender>
   <signal>valid(bool)</signal>
   <receiver>_post</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
